pages/smart-account-guards.mdx

```diff
@@ -4,7 +4,7 @@ import { Callout } from 'nextra/components'
 
 <Callout type='info' emoji='ℹ️'>
   Safe Guards are introduced with [Safe contracts version
-  1.3.0](https://github.com/safe-global/safe-contracts/blob/v1.3.0/CHANGELOG.md).
+  1.3.0](https://GitHub.com/safe-global/safe-contracts/blob/v1.3.0/CHANGELOG.md).
```

Comment:
I'm not sure this is the best solution. Links should be valid as they are

Reply:
Could not find any other way to solve the error without deactivating a major part of vale